flex_container_orchestrator/domain/aggregator_flexpart.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+
+import argparse
+import datetime
+import json
+import logging
+import sqlite3
+import sys
+from typing import List, Optional, Set, Tuple
+
+from flex_container_orchestrator import CONFIG
+
+
+def parse_arguments() -> argparse.Namespace:
+    """
+    Parse command-line arguments.
+
+    Returns:
+        argparse.Namespace: Parsed arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Launch Flexpart after processing checks."
+    )
+    parser.add_argument("--date", required=True, help="Date in YYYYMMDD format")
+    parser.add_argument("--time", required=True, help="Time in HH format")
+    parser.add_argument(
+        "--step", required=True, help="Step identifier (lead time in hours)"
+    )
+    parser.add_argument("--db_path", required=True, help="Path to the SQLite database")
+    return parser.parse_args()
+
+
+def connect_db(db_path: str) -> sqlite3.Connection:
+    """
+    Establish a connection to the SQLite database.
+
+    Args:
+        db_path (str): Path to the SQLite database.
+
+    Returns:
+        sqlite3.Connection: SQLite connection object.
+    """
+    try:
+        conn = sqlite3.connect(db_path)
+        logging.info("Connected to SQLite database at %s.", db_path)
+        return conn
+    except sqlite3.Error as e:
+        logging.error("SQLite connection error: %s", e)
```

Comment:
```suggestion
        logger.error("SQLite connection error: %s", e)
```
also in the rest of the file